__tests__/app.test.js

```diff
@@ -1,37 +1,45 @@
 const app = require('../app')
-const supertest = require('supertest')
+const request = require('supertest')
+const data = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
 
-// beforeAll(done => {
-//     done()
-//   })
 
-// afterAll(done => {
-//     app.close();
-//     done();
-// });
+beforeAll(done => { 
+seed({data})
+    done()
+})
+
 
 describe('Topics endpoints', ()=>{
-    test('GET /api/topics should return a json containing all topics with status 200', async ()=>{
-        // const apiRes = {topics: [ { slug: 'coding', description: 'Code is love, code is life' },
-        // { slug: 'football', description: 'FOOTIE!' },
-        // { slug: 'cooking',
-        //   description: 'Hey good looking, what you got cooking?' }]}
-        
-     supertest(app).get('/api/topics').expect(200)
+    test('GET /api/topics should return a json containing all topics with status 200', ()=>{    
+    request(app).get('/api/topics').expect(200)
```

Comment:
It's crucial that if you've refactored your tests so that they are NOT async functions, you return the whole promise chain - otherwise this test will pass regardless of the outcome from your server